core/src/main/resources/application.yml

```diff
@@ -699,7 +699,7 @@ cb:
   paywall.url: "https://archive.cloudera.com/p/cdp-public/"
   nodestatus:
     connectionTimeoutMs: 5000
-    readTimeoutMs: 5000
+    readTimeoutMs: 30000
```

Comment:
hope it won't be too large as during node status checks we are using the same values, and there are 4-5 different request for 1 cluster

Reply:
I increased the connection timeout to 15 sec.